### PR TITLE
Makefile Updates for Integration Site

### PR DESCRIPTION
**Pull request by madelinezec**
Here is a short spec about the integration site: https://docs.google.com/document/d/1AhM3TpB7mv_cqLgqj57KpmCahDahWSCmXLqmsc5Oc6w/edit?usp=sharing

This PR represents the makefile side changes required to implement the integration site. It is important to note that we do not support devhub or cloud docs on the integration site because neither publish to the docs.mongodb.com domain nor do they have prefixes (or in this case, suffixes that come after the domain).

This spreadsheet shows job urls and integration site links for every repo. Not every single branch of every repo was tested (sparky and bi-connector have many, many). If it is required that we test every single branch, I can set aside time to do that. 
https://docs.google.com/spreadsheets/d/1ZpRJpyeMANTanjGdbgNkp7Ik0y5fNsHc-B0887pelTs/edit?usp=sharing

I will squash before merging!

### CODE REVIEW

File: makefiles/Makefile.cloud-docs-osb [hunk 1]
```diff
@@ -1,9 +1,16 @@
 GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
 USER=$(shell whoami)
 STAGING_URL="https://docs-mongodborg-staging.corp.mongodb.com"
-PRODUCTION_URL="https://docs.mongodb.com"
 STAGING_BUCKET=docs-mongodb-org-staging
-PRODUCTION_BUCKET=docs-atlas-osb-prod
+
+ifeq ($(SNOOTY_INTEGRATION),false)
+  PRODUCTION_URL="https://docs.mongodb.com"
+  PRODUCTION_BUCKET=docs-atlas-osb-prod;
```
Comment (i80and) [inline makefiles/Makefile.cloud-docs-osb:8]: Are these semicolons intentional?

File: makefiles/Makefile.docs-bi-connector [hunk 2]
```diff
@@ -22,7 +29,7 @@ next-gen-html: get-build-dependencies
 	else \
 		exit 0; \
 	fi
-	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty ${REPO_DIR}; \
+	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty ${REPO_DIR};
```
Comment (i80and) [inline makefiles/Makefile.docs-bi-connector:32]: Semicolon is unnecessary if you're not doing a line continuation

Comment (i80and) [inline makefiles/Makefile.docs-bi-connector:32]: shell variables should be wrapped in double-quotes: e.g.
```
rsync -az --exclude '.git' "${REPO_DIR}/../../snooty" "${REPO_DIR}"
```

File: makefiles/Makefile.docs-charts [hunk 3]
```diff
@@ -23,23 +29,23 @@ next-gen-html:
...
+	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty ${REPO_DIR};
+	cp ${REPO_DIR}/.env.production ${REPO_DIR}/snooty;
 	cd snooty; \
 	echo "GATSBY_SITE=${PROJECT}" >> .env.production; \
 	npm run build; \
 	cp -r "${REPO_DIR}/snooty/public" ${REPO_DIR};
   
 next-gen-stage: ## Host online for review
-	mut-publish public ${STAGING_BUCKET} --prefix="${MUT_PREFIX}" --stage ${ARGS}; \
+	mut-publish public ${STAGING_BUCKET} --prefix="integration/${MUT_PREFIX}" --stage ${ARGS}; \
```
Comment (i80and) [inline makefiles/Makefile.docs-charts:40]: Is staging now under `integration/` for charts?

Comment (madelinezec) [reply makefiles/Makefile.docs-charts:40]: doh! thank you for catching this!

**Review by i80and — APPROVED** [review]

**Pull request merged**